app/controllers/wissles_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class WisslesController < ApplicationController
+  before_action :authenticate!
+
+  # GET /wissles
+  def index
+    @resource = Wissle.all
+
+    render :index
+  end
+
+  # POST /
+  def create
+    @resource = Wissle.new wissle_params
+
+    if @resource.save
+      render :show, :status => :created
+    else
+      render :show, :status => :unprocessable_entity
+    end
+  end
+
+  # GET /:id
+  def show
+    @resource = Wissle.find params[:id]
+
+    render :show
+  end
+
+  # PUT/PATCH /:id
+  def update
```

Comment:
If I'm not mistaken, we don't allow people to edit their Wissles, only delete them. So you should replace this method with a `:destroy` method.